### PR TITLE
Deploy Telegram bot to test and demo envs

### PR DESCRIPTION
**Pull request by antonyshchenko**
Also:
- fixed broken trade submission (veryfing chain id was not set properly)
- added fault-tolerance to bot websockets
- added pending deposits to bot session state in DB so that they don't get lost after restart
- moved remnants of messaging from `BotSessionCurrentWallet` to `BotSession` so it is consolidated in one place and `BotSessionCurrentWallet` does not depend on `BotSession`

Test env bot: https://t.me/chainring_test_bot
Demo env bot: https://t.me/chainring_demo_bot

### CODE REVIEW

File: scripts/envs/demo.yml [hunk 1]
```diff
@@ -56,6 +62,14 @@ services:
     secrets:
       DB_CREDENTIALS: arn:aws:secretsmanager:us-east-2:851725450525:secret:rds!cluster-d567d2ec-6ebb-4c6d-b48a-529eb09febfb-KMOVrv
       SLACK_ERROR_REPORTING_APP_TOKEN: arn:aws:secretsmanager:us-east-2:851725450525:secret:slack-error-reporter-token-74wImX
+  telegrambot:
+    image: 851725450525.dkr.ecr.us-east-2.amazonaws.com/telegrambot
+    count: 1
+    is_essential: "true"
```
Comment (imykolenko) [inline scripts/envs/demo.yml:68]: is there a reason for holding page when deploying telegram bot?

Comment (antonyshchenko) [reply scripts/envs/demo.yml]: no, thanks, updated

**Review by imykolenko — APPROVED** [review]
👍

**Review by bholzman — APPROVED** [review]
Looks great, nice job!

**Pull request merged**